content/en-us/reference/engine/classes/AudioLimiter.yaml

```diff
@@ -29,8 +33,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, unclamped.
+      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
```

Comment:
We could add a sentence giving the range for this parameter

Reply:
Oh good call
```suggestion
      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
      This value ranges from -12 to 0.
```